app/models/menu_items_menu.rb

```diff
@@ -0,0 +1,20 @@
+class MenuItemsMenu < ActiveRecord::Base
```

Comment:
Not a big deal, but we could have a better model/table name maybe like `Inventory`.